### DISCUSSION TITLE
RSSI fluctuation

### DISCUSSION THREAD

**Discussion by ImTandoff** [sandeepmistry/arduino-LoRa#609] [Q&A] [answered]
Hello everyone. I am developing a private communication with LoRa. In the currently stage of the project is importante to analyze the state of the comunnication, RSSI, and SNR indicator.

Plotting the RSSI and SNR indicator for a short communication, the RSSI were fluctuating a lot when the prototype were moved as you can see in the picture below. 

Img url : https://www.linkpicture.com/q/PLOTTING.png

Does anyone know why this happens. Because in the stage of the project there is a need to certify the relability of the communication.
The modulation parameters for the test were:

#define TX_POWER 20
#define SF 12
#define BANDWIDTH 125E3
#define SYNCWORD 0xF3
#define CODERATE 1
#define PREAMBLELENGTH 10
#define LNA_GAIN 6

**Comment by IoTThinks** [comment, marked as answer]
Likely you put them so near to each other.
Put them at least 5 meters from each other.
